WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/ExpandableBottomSheet.swift

```diff
@@ -79,7 +79,7 @@ struct ExpandableBottomSheet<AlwaysVisibleContent, ExpandableContent>: View wher
             // Always visible content
             alwaysVisibleContent()
                 .trackSize(size: $fixedContentSize)
-                .onChange(of: fixedContentSize, perform: { [fixedContentSize] _ in
+                .onChange(of: fixedContentSize) {
```

Comment:
It doesn't seem we need the capture list here: `.onChange(of: fixedContentSize)` already re-runs the closure every time `fixedContentSize` changes, so capturing it when created serves no purpose as it will always have the latest value when the closure runs anyway